frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -20,14 +19,14 @@
 
   $: user = data.user;
   $: project = data.project;
-  $: _project = project as NonNullable<typeof project>;
+  $: _project = project ;
```

Comment:
🤔

Reply:
I think maybe project is never null now?